app/components/configure-minted-asset/ConfigureMintedAsset.tsx

```
@@ -0,0 +1,181 @@
+    );
+  };
+  return (
+    <div className="fixed inset-0 top-0 h-screen overflow-y-auto bg-black bg-opacity-90">
+      <div className="relative top-44 mx-auto w-full max-w-2xl rounded-lg bg-white p-8 shadow dark:bg-slate-800">
+        <button
+          onClick={() => {
+            props.setAddClaimPhase(false);
+            props.toggleModal();
+          }}
+          className="absolute right-4 top-4 mb-4 text-gray-500 hover:text-gray-600 focus:outline-none dark:hover:text-gray-200"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)